README.md

```diff
@@ -19,8 +38,17 @@ corepack yarn
 corepack yarn dev
 ```
 
-This command starts a local development server and opens up a browser window.
-Most changes are reflected live without having to restart the server.
+This command builds **/uppy/docs** and **/uppy.io**, and shows them in a
+browser. Most changes are reflected live without having to restart the server.
```

Comment:
I think the previous sentence was clearer. Can you explain what you don't like about it?